app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsPresenter.kt

```diff
@@ -137,6 +142,19 @@ class SkillDetailsPresenter(skillDetailsFragment: SkillDetailsFragment) : ISkill
         }
     }
 
+    override fun reportSendError(throwable: Throwable) {
+        Timber.d(throwable)
+        skillDetailsView?.updateSkillReportStatus("Coudn't report the skill")
+    }
+
+    override fun reportSendSuccess(response: Response<ReportSkillResponse>) {
+        if (response.code() == 422) {
+            skillDetailsView?.updateSkillReportStatus("Skill Already Reported")
```

Comment:
Use string resource. Why capital 'A' and 'R'?

Reply:
same